tests/test_schema_generation.py

```diff
@@ -53,3 +54,14 @@ class TestStandardPythonSchemaGeneration(object):
     def test_standard_handling(self):
         assert get_input_schema(standard_py_func) == self.standard_sample_input_schema
         assert get_output_schema(standard_py_func) == self.standard_sample_output_schema
+
+
+class TestNestedSchemaGeneration(TestCase):
```

Comment:
What does extending TestCase add? Considering we haven't had it before, just curious

Reply:
After extending `TestCase` we can now use [assertCountEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual), this is differ from ordinary `assert` because it ignores the order in sequence. After Python3.6, dict keys maintain their insertion order by default ([here](https://stackoverflow.com/questions/1867861/how-to-keep-keys-values-in-same-order-as-declared)), `assertCountEqual` will help us pass integration test under Python3.5 since we do not care the order of keys in `required` field of schema